### PR TITLE
Added sorting method to match real microscope

### PR DESCRIPTION
**Pull request by benblamey**

### CODE REVIEW

File: simulator_no_flask.py [hunk 1]
```diff
@@ -75,6 +76,13 @@ def get_files(image_directory_path, period, binning, color_channel_filter, send_
         files = {filename: file_info for filename, file_info in files.items()
                  if file_info['color_channel'] in color_channel_filter}
 
+    # sort the image stream to match the real microscope
+
+    files = OrderedDict(sorted(files.items(), key=lambda file_info: (file_info[1]['time_point_number'],
+                                                                    file_info[1]['well'],
+                                                                    file_info[1]['imaging_point_number'],
+                                                                    file_info[1]['color_channel'])))
+
```
Comment (benblamey) [inline simulator_no_flask.py]: looks great - has this been tested?

Comment (benblamey) [reply simulator_no_flask.py]: Please fix the whitespace also!

Comment (Hakanwie) [reply simulator_no_flask.py]: It has been tested locally but not in the streaming setting, but that shouldn't make much difference

**Comment by benblamey** [comment]
Lets merge and test with the proper dataset

**Pull request merged**